build.gradle

```diff
@@ -13,6 +13,7 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 repositories {
+    mavenLocal()
```

Comment:
Probably don't need this now with github.io working again?

Reply:
True. Ta.